modules/clair-disconnected.adoc

```diff
@@ -1,50 +1,67 @@
-[[clair-disconnected]]
-= Configuring Clair for Disconnected Environments
+:_content-type: CONCEPT
+[id="configuring-clair-disconnected-environments"]
+= Configuring Clair for disconnected environments
 
-Clair utilizes a set of components called Updaters to handle the fetching and parsing of data from various vulnerability databases.  These Updaters are set up by default to pull vulnerability data directly from the internet and work out of the box.  For customers in disconnected environments without direct access to the internet this poses a problem.  Clair supports these environments through the ability to work with different types of update workflows that take into account network isolation.  Using the `clairctl` command line utility, any process can easily fetch Updater data from the internet via an open host, securely transfer the data to an isolated host, and then import the Updater data on the isolated host into Clair itself.
+Clair uses a set of components called `Updaters` to handle the fetching and parsing of data from various vulnerability databases. `Updaters` are set up by default to pull vulnerability data directly from the internet and work for immediate use. For customers in disconnected environments without direct access to the internet, this poses a problem. Clair supports these environments by working with different types of update workflows that take into account network isolation. Using the `clairctl` command line utility, any process can easily fetch `Updater` data from the internet by using an open host, securely transfer the data to an isolated host, and then import the `Updater` data on the isolated host into Clair itself. 
 
-The steps are as follows.
+[NOTE]
+====
+Clair enrichment data is currently unsupported in disconnected environments. 
```

Comment:
At the moment the only enrichment data is CVSS data, would it be possible to mention CVSS as well for people who'd be looking for that?

Reply:
@crozzy Do you want it put here, or in the section above? https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/manage_red_hat_quay/clair-intro2#clair-cve

If you want it here, I might say... 

"Currently, Clair enrichment data is CVSS data. Enrichment data is currently unsupported in disconnected environments. "

If I in the other section, I could maybe say... 

"With Clair v4.2,  CVSS enrichment data is now viewable in the Quay UI."

WDYT?

Reply:
Yeah, I think that makes sense and will be best for users searching either "cvss" or "enrichment"